### PR TITLE
Improve docs (README and Go Doc comments)

### PR DESCRIPTION
**Pull request by bplunkett-stripe**
### Description
[//]: # (A clear and concise description of the purpose of this Pull Request. What is being changed? Include any relevant background for this change.)
Improve README and Go Doc comments

### Motivation
[//]: # (Why you made these changes. Link to any relevant issues.)
Make it easier for folks to integrate with the library

### Testing
[//]: # (Describe how you tested these changes)
None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,15 +41,15 @@ pg-schema-diff plan --dsn "postgres://postgres:postgres@localhost:5432/postgres"
 - Online index replacement
 
 # Install
-## Library
-```bash
-go get -u github.com/stripe/pg-schema-diff
-````
 ## CLI
 ```bash
```
Comment (bplunkett-stripe) [inline README.md:45]: People are probably more interested in the CLI than the library, so let's put that first

File: README.md [hunk 2]
```diff
@@ -76,6 +76,48 @@ Apply the schema. Any hazards in the generated plan must be approved
 pg-schema-diff apply --dsn "postgres://postgres:postgres@localhost:5432/postgres" --schema-dir schema --allow-hazards INDEX_BUILD
 ```
 
+# Using Library
+Docs to use the library can be found [here](https://pkg.go.dev/github.com/stripe/pg-schema-diff). Check out the CLI
```
Comment (alexaub-stripe) [inline README.md:80]: thought: Link to the CLI source?

**Review by alexaub-stripe — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (bplunkett-stripe) [reply README.md]: Good point. Added!

**Review by alexaub-stripe — APPROVED** [review]

**Pull request merged**